src/main/java/com/adamk33n3r/runelite/watchdog/ui/alerts/InventoryAlertPanel.java

```diff
@@ -19,9 +19,14 @@ protected void build() {
             })
             .addIf(
                 panel -> panel.addRegexMatcher(this.alert, "Enter the name of the item to trigger on...", "The name to trigger on. Supports glob (*)")
-                    .addSpinner("Quantity", "The quantity of item to trigger on, use 0 for every time", this.alert.getItemQuantity(), this.alert::setItemQuantity),
+                    .addSpinner("Quantity", "The quantity of item to trigger on, use 0 for every time", this.alert.getItemQuantity(), this.alert::setItemQuantity, 0, Integer.MAX_VALUE, 1),
                 () -> this.alert.getInventoryAlertType() == InventoryAlert.InventoryAlertType.ITEM
             )
+            .addIf(
+                panel -> panel.addRegexMatcher(this.alert, "Enter the name of the item to trigger on...", "The name to trigger on. Supports glob (*)")
+                    .addSpinner("Change", "The quantity change of the item (in one tick) to trigger on", this.alert.getItemQuantity(), this.alert::setItemQuantity),
```

Comment:
Oh I didn't realize the defaults were so restricted. I assumed they'd be -Inf, Inf.

When is that default change going in? What change should I make in this PR?

Reply:
Actually, I just realized. If the defaults are changed to `0...Inf` then this won't work as expected since it should be `-Inf...Inf`. I'll update here.

Reply:
Ya the `-99...99` is from the stat change alert that just got copied over I guess and never fixed. It'll go in soon, probably after I merge your stuff. I could be convinced to make the defaults `0...Inf` or `-Inf...Inf`. And by Inf you mean `Integer.MAX_VALUE`?

Reply:
> And by Inf you mean Integer.MAX_VALUE?

Yup